README.md

```diff
@@ -38,16 +33,33 @@ Templates planned to be included:
 
 	$ nic.pl -p "package.name" -n "ProjectName" -u "User name" --nic template.nic.tar
 
+## Building your own template
+
+- Copy a preexisting directory from ./ios/ OR run denicify.pl on a preexisting .nic.tar
+- Rename the resulting directory
+- Edit its contents as desired
+- Run nicify.pl on it
+	- Note: the resulting template can be placed in nic/templates/ for use with the NIC (i.e., nic.pl)
```

Comment:
I think this should be `$THEOS/templates`.
I see there's also `$THEOS/vendor/nic/templates`, which I'm guessing is what you're referencing here.

Reply:
Oop, thanks! Could've sworn there was a reason I left it like that, but nothing is coming to mind.